README.md

```diff
@@ -22,7 +22,10 @@ Sample Set #1                    |  Sample Set #2
 - Select Pokemon by name or by index number
 - Ability to change the Desktop Wallpaper & the Terminal background
 - Internal search system for finding Pokemon
-- Supports iTerm2, Terminology & Tilix
+- Supports iTerm2, Terminology, Tilix, Terminator and XFCE4Terminal (see notes below)
+
+> Terminator: Config file must exist. This is done by opening the settings dialog, setting a dummy background image  and closing it.  
+> XFCE4Terminal: Due to the way terminator read its config file, it is *impossible* to update the background image as soon as the command ends. You must either restart terminator (yaiks) or open the settings dialog and closing (no need to touch anything else).
```

Comment:
Are you sure about this too? Where did you read this? Worst case scenario, can't we restart, or ask to?